packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -243,13 +246,24 @@ describe("normalizeAndValidateConfig()", () => {
 
         expect(config).toEqual(expect.objectContaining(expectedConfig));
         expect(diagnostics.hasErrors()).toBe(false);
-        expect(diagnostics.hasWarnings()).toBe(false);
+        expect(diagnostics.hasWarnings()).toBe(true);
+        if (!isWindows) {
+          expect(diagnostics.renderWarnings()).toMatchInlineSnapshot(`
+          "Processing wrangler configuration:
+            - DEPRECATION: \\"site.entry-point\\":
+              Delete the \`site.entry-point\` field, then add the top level \`main\` field to your configuration file:
+              \`\`\`
+              main = \\"my-site/index.js\\"
+              \`\`\`"
+        `);
```

Comment:
So wait...there's no equivalent check for windows? I suppose this can also go in the issue i mentioned